lib/ui/puyo/touch.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-env browser */
```

Comment:
For clarity I think this file should be called `touch-events.js` as it only contains events.